src/java.com/executors/thread/pool/ThreadPoolApplication.java

```diff
@@ -0,0 +1,16 @@
+package executors.thread.pool;
+
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadPoolExecutor;
+
+public class ThreadPoolApplication {
+    public static void main(String[] args) {
+        ThreadPoolExecutor executor = (ThreadPoolExecutor) Executors.newFixedThreadPool(2);
```

Comment:
You don't need cast here. it could be rewritten.
`ExecutorService executor = Executors.newFixedThreadPool(2);`